service/controller/app/v1/resource/clients/create.go

```diff
@@ -21,5 +21,10 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		return microerror.Mask(err)
 	}
 
+	err = r.addHelmClientToContext(ctx, cr)
```

Comment:
Why don't we add the client in the method that already exists? There are clients added to the context already.

Reply:
Not sure which clients in the method you are meaning, you mean `helmClient`?

Reply:
There is `addClientsToContext` already. It adds the `g8sClient` and the `k8sClient` to the context. Why do we need another method to add the helm client to the context? Why can't we add the helm client to the context in `addClientsToContext`?

Reply:
I thought I need to differentiate two clients and helm client, but seems that's not the good practice, I'm merging two functions as you said, Thanks a lot!

Reply:
@xh3b4sd PTAL :)